api/handlers.go

```diff
@@ -19,6 +20,22 @@ import (
 	"gitlab.com/NebulousLabs/errors"
 )
 
+const (
+	// userLimitCacheTTL is the TTL of the entries in the userLimitCache.
+	userLimitCacheTTL = time.Hour
+)
+
+var (
+	// userLimitCacheEntry is an in-mem cache that maps from a user's sub to
+	// the data we care about.
+	userLimitCache   = map[string]userLimitCacheEntry{}
+	userLimitCacheMu sync.Mutex
+	// userLimits is a caching entry to avoid doing the same static operation
+	// over and over.
+	userLimits   []TierLimitsPublic
```

Comment:
This seems like something that could just be a `staticUserLimits` field in the `API` which is pulled from the db when you call `New(...) (*API, error)`